pkgs/applications/virtualization/ddev/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, bash
+, curl
+, docker
+, docker-compose
+, fetchFromGitHub
+, git
+, go
+, pkgs
+, stdenv
+, testers
+, ddev }:
+
+
+let
+  pname = "ddev";
+  version = "1.21.6";
+in
+
+stdenv.mkDerivation {
+  name = pname;
+  version = version;
+  src = fetchFromGitHub {
+    owner = "ddev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Wjg0Yxo/ulY6R6hhUMFvNZUSwpXENmAHU7GPbgdw7tw=";
+  };
+  buildInputs = [ bash curl git go ];
```

Comment:
Please patch that the build requires a git clone. We can replace the values returned by git with dummy values or version.

```
build flags: SHELL=/nix/store/zlf0f88vj30sc7567b80l52d19pbdmy2-bash-5.2-p15/bin/bash build
fatal: not a git repository (or any of the parent directories): .git
make: *** [Makefile:96: .gotmp/bin/darwin_arm64/mkcert] Error 6
``` 

Also please split the packages for cross compilation into nativeBuildInputs and buildInputs.